monai/networks/nets/segresnet_ds.py

```diff
@@ -387,7 +388,7 @@ def is_valid_shape(self, x):
         a = [i % j == 0 for i, j in zip(x.shape[2:], self.shape_factor())]
         return all(a)
 
-    def _forward(self, x: torch.Tensor) -> torch.Tensor | list[torch.Tensor]:
+    def _forward(self, x: torch.Tensor) -> Union[None, torch.Tensor, list[torch.Tensor]]:
```

Comment:
Hi @wyli , I tested with `Union[torch.Tensor, list[torch.Tensor]]` and it works. Could I know that what is reason here to add `None`?

Reply:
same for `ds_loss`, may also no need to add `None` in type hint. Or will it impact other test functions?
I just run `tests/test_ds_loss.py` and `tests/test_segresnet_ds.py`

Reply:
The changes are needed for torch<2 and python>3.8, for example https://github.com/Project-MONAI/MONAI/actions/runs/4928134014/jobs/8806104746?pr=6498

Reply:
thanks!